### PR TITLE
Update sessions.md

### PR DESCRIPTION
**Pull request by sebastjan-hribar**
This update is about short instruction for sharing session values.

### CODE REVIEW

File: content/actions/sessions.md [hunk 1]
```diff
@@ -61,3 +61,6 @@ module Web
   end
 end
 ```
+### Sharing session values between applications
+
+To share session values, defined in one application, we must provide the same session secret to all the applications where we need those values.
```
Comment (adam12) [inline content/actions/sessions.md:66]: Would it be worth duplicating the example code block above to this section, and then displaying two application classes with the same session key set for both?

Comment (sebastjan-hribar) [reply content/actions/sessions.md:66]: Thank you for the suggestion, I've made the changes.

**Comment by adam12** [comment]
Sorry for the delay on this. Are you able to rebase @sebastjan-hribar?

**Comment by adam12** [comment]
Closing in favour of #104

**Pull request closed without merging**